backend/test/address.js

```diff
@@ -0,0 +1,77 @@
+/*const supertest = require("supertest");
```

Comment:
I guess this file will be added by Caching PR. So, I think these changes should not be added here.